hikari/api/rest.py

```diff
@@ -780,6 +817,43 @@ async def create_message(
             once before being able to use this endpoint for a bot.
         """  # noqa: E501 - Line too long
 
+    @abc.abstractmethod
+    async def crosspost_message(
```

Comment:
could we call this `create_crosspost` instead?

I know `crosspost` is a verb in this context, but we also use it as a noun elsewhere, which may be slightly more ambiguous to the reader.

Reply:
~~We could even call this `create_announcement` tbh if we want to match the lingo that the client uses.~~
Nevermind, seems like Discord have just brought the idea back again after letting it die silently last year I guess.

Reply:
Should `create_crosspost` not be plural?

Reply:
i don't overly like "crosspost" as a word, but crossposts would be fine.

if you are concerned about ambiguity, just alias it. We already do this for `ban_user/ban_member` and `unban_user/unban_member` for the same reason.